beacon-chain/blockchain/head.go

```diff
@@ -24,31 +24,24 @@ import (
 	"go.opencensus.io/trace"
 )
 
-// UpdateAndSaveHeadWithBalances updates the beacon state head after getting justified balanced from cache.
-// This function is only used in spec-tests, it does save the head after updating it.
-func (s *Service) UpdateAndSaveHeadWithBalances(ctx context.Context) error {
```

Comment:
I would prefer us to not change the signature. Instead, just log the error as it returns in:
beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go